contracts/sol6/Dao/IKyberStaking.sol

```diff
@@ -27,7 +27,7 @@ interface IKyberStaking is IEpochUtils {
 
     function withdraw(uint256 amount) external;
 
-    function getStakerDataForPastEpoch(address staker, uint256 epoch)
+    function getStakerRawData(address staker, uint256 epoch)
```

Comment:
better add 1 comment here 
that this function returns valid values only for a staker that didn't delegate

Reply:
Not exactly, only return valid value if staker has done at least 1 action at previous or that epoch

Reply:
but its 99% that if he delegated.
he has done no action.

so pool operator shouldn't be using this by design...

Reply:
yeah, if he didn't delegate, he should have voted for some campaigns

Reply:
Added more comment there, please help to re-check

Reply:
OK
I think should its crystal clear now - for pool operators